src/Microsoft.NetCore.Analyzers/RulesMissingDocumentation.md

```diff
@@ -2,15 +2,12 @@
 
 Rule ID | Missing Help Link | Title |
 --------|-------------------|-------|
-CA1830 | https://docs.microsoft.com/visualstudio/code-quality/ca1830 | Prefer strongly-typed Append and Insert method overloads on StringBuilder. |
-CA1831 | https://docs.microsoft.com/visualstudio/code-quality/ca1831 | Use AsSpan or AsMemory instead of Range-based indexers when appropriate |
-CA1832 | https://docs.microsoft.com/visualstudio/code-quality/ca1832 | Use AsSpan or AsMemory instead of Range-based indexers when appropriate |
-CA1833 | https://docs.microsoft.com/visualstudio/code-quality/ca1833 | Use AsSpan or AsMemory instead of Range-based indexers when appropriate |
-CA1834 | https://docs.microsoft.com/visualstudio/code-quality/ca1834 | Consider using 'StringBuilder.Append(char)' when applicable. |
-CA1835 | https://docs.microsoft.com/visualstudio/code-quality/ca1835 | Prefer the 'Memory'-based overloads for 'ReadAsync' and 'WriteAsync'. |
-CA1836 | https://docs.microsoft.com/visualstudio/code-quality/ca1836 | Prefer IsEmpty over Count |
+CA1834 | https://docs.microsoft.com/visualstudio/code-quality/ca1834 | Consider using 'StringBuilder.Append(char)' when applicable |
+CA1837 | https://docs.microsoft.com/visualstudio/code-quality/ca1837 | Use 'Environment.ProcessId' |
+CA1838 | https://docs.microsoft.com/visualstudio/code-quality/ca1838 | Avoid 'StringBuilder' parameters for P/Invokes |
```

Comment:
FYI elinor-fung - the analyzers you recently added need to be documented as per the Documentation requirements at https://github.com/dotnet/roslyn-analyzers/blob/master/GuidelinesForNewRules.md

Reply:
Tagging @elinor-fung for real

Reply:
Yep - doc for CA1838 is in PR: https://github.com/MicrosoftDocs/visualstudio-docs-pr/pull/6959

Reply:
THanks!